create-certificates.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -x
+# Create certificates for registry authentication
+rm -rf ../certs
+mkdir ../certs
+openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes  -keyout ../certs/registry.key -out ../certs/registry.crt -subj "/CN=registry"  -addext "subjectAltName=DNS:cnftest-local.redhat.com,IP:127.0.0.1"
+openssl x509 -in ../certs/registry.crt -out ../certs/registry.pem -outform PEM
```

Comment:
update readme to have openssl as dependency

Reply:
it is part of the ubuntu image. The operator scrips only work on Ubuntu (matches Github). Later we can optimize the test to just deploy the operator, then it should be agnostic of platform.